docs/device-control.rst

```diff
@@ -0,0 +1,66 @@
+Device Control
+==============
+
+Using the Carbon Black Cloud SDK, you can retrieve information about USB devices used in your organization, and manage
+the blocking of such devices from access by your endpoints.
+
+Retrieving the List of Known USB Devices
+----------------------------------------
+
+Using a query of the `USBDevice` object, you can see which USB devices have been used on any endpoint in your
+organization.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.endpoint_standard import USBDevice
+    >>> query = api.select(USBDevice).where('1')
```

Comment:
Only one comment can we use `*` instead of `1` for the where statement as I believe they will work the same and will be more indicative that we are trying to match all. Please apply to all locations

Reply:
Actually, no it doesn't, and I found this out the hard way while running through the operations on the command line to create the transcripts you see here.

Reply:
Transcript for proof:
```
C:\Users\abowersox>python
Python 3.8.1 (tags/v3.8.1:1b293b6, Dec 18 2019, 23:11:46) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from cbc_sdk import CBCloudAPI
>>> api = CBCloudAPI(profile='test03-dc')
Security warning: Windows file access to CBC SDK credentials is inherently insecure
A future version of CBC SDK will default to using a registry-based credentials provider.
>>> from cbc_sdk.endpoint_standard import USBDevice
>>> query = api.select(USBDevice).where('*')
>>> for usb in query:
...   print(f"{usb.vendor_name} {usb.product_name} {usb.serial_number} {usb.status}")
...
>>> query = api.select(USBDevice).where('1')
>>> for usb in query:
...   print(f"{usb.vendor_name} {usb.product_name} {usb.serial_number} {usb.status}")
...
SanDisk Ultra 4C531001331122115172 UNAPPROVED
SanDisk Cruzer Dial 4C530000110722114075 UNAPPROVED
PNY USB 2.0 FD 07189613DD84E242 UNAPPROVED
USB Flash Disk FBI1305031200020 APPROVED
>>> ^Z
```
Note that the first query (with `*` returns nothing, but the second query, with `1`, works as expected.

Reply:
Thanks for the clarification! Resolving thread